content/en/news/releases/1.5.x/announcing-1.5/_index.md

```diff
@@ -15,4 +15,106 @@ We are pleased to announce the release of Istio 1.5!
 
 {{< relnote >}}
 
-TBD
+We've made it simpler to
+install and run Istio by consolidating the control plane components into a
+single binary; we've introduced a powerful and fast new extension model for
+proxy servers across the industry, and we've continued to improve usability,
+security, telemetry and traffic control.
+
+After a year of amazing growth and learning, we have packed more in this release
+than any since 1.1.  A year ago we decided to move to quarterly releases, and
+we are happy to report that this is the fifth consecutive time we have met that
+goal. Users are getting more new functionality faster than ever!
+
+Here’s some of what’s coming to you in today's release:
+
+## Introducing `istiod`
+
```

Comment:
We need a diagram here. This will be the canonical diagram that everyone will use

Reply:
Maybe a link from here to the updated architecture diagram being worked on in #6711. Btw, we could use some input on how exactly the istiod diagram should look.

Reply:
Also, I don't think we have official guidance on Istiod vs istiod but I understand we use Istiod to talk about the component and istiod when talking about the deployment. @frankbu, correct me if I'm wrong. I know I've asked this question many times myself.

Reply:
There is going to be a post dedicated to istiod in the next week...think the diagram can wait for that?

Reply:
@oaktowner will the diagram in the post next week also be a suitable replacement for the architecture diagram in the op guide (https://preliminary.istio.io/docs/ops/deployment/architecture/)? At a minimum, I think the two diagrams should be consistent in style and content.

Reply:
Yes, standardized diagrams are great. But are you OK with this post going out without one and instead saving that for the post dedicated to istiod?